File: apps/s-2-admin/src/api/schedule/ScheduleWhereInput.ts
import { IntFilter } from "../../util/IntFilter";
import { UserWhereUniqueInput } from "../user/UserWhereUniqueInput";
import { StringFilter } from "../../util/StringFilter";
import { StringNullableFilter } from "../../util/StringNullableFilter";
import { EventTypeListRelationFilter } from "../eventType/EventTypeListRelationFilter";
import { AvailabilityListRelationFilter } from "../availability/AvailabilityListRelationFilter";

export type ScheduleWhereInput = {
  id?: IntFilter;
  user?: UserWhereUniqueInput;
  name?: StringFilter;
  timeZone?: StringNullableFilter;
  eventType?: EventTypeListRelationFilter;
  availability?: AvailabilityListRelationFilter;
};

File: apps/cactus-demo-dotnet/src/APIs/Schedule/Dtos/ScheduleWhereInput.cs
namespace CactusDemoDotnet.APIs.Dtos;

public class ScheduleWhereInput
{
    public int? Id { get; set; }

    public int? User { get; set; }

    public string? Name { get; set; }

    public string? TimeZone { get; set; }

    public List<int>? EventType { get; set; }

    public List<int>? Availability { get; set; }
}
